packages/infra/heroku/grant-dagcargo.sql

```diff
@@ -0,0 +1,2 @@
+GRANT USAGE ON SCHEMA public TO dagcargo;
+GRANT SELECT ON public.aggregate, public.aggregate_entry, public.auth_key, public.content, public.deal, public.pin, public.upload, public.user TO dagcargo;
```

Comment:
Could simplify this by granting read on all tables in the public schema, and for all new table too
```suggestion
GRANT SELECT ON ALL TABLES IN SCHEMA public TO dagcargo;
ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT SELECT ON TABLES TO dagcargo;
```

Reply:
this is what i used for managing perms on fil-flow